sdcm/utils/remote_logger.py

```diff
@@ -117,6 +117,18 @@ def stop(self, timeout=None):
 
 
 class SSHScyllaSystemdLogger(SSHLoggerBase):
+
+    def _wait_ssh_up(self, *args, **kwargs):
+        super()._wait_ssh_up(*args, **kwargs)
+        node = self.node
+        if node.remoter.sudo("which python3", ignore_status=True).failed:
+            if node.distro.is_rhel_like:
+                node.remoter.sudo("yum install -y python3", retry=3)
+            elif node.distro.is_sles:
+                node.remoter.sudo("zypper update -y python3", retry=3)
+            else:
+                node.remoter.sudo('apt-get install -y python3', retry=3)
```

Comment:
```suggestion
        if node.remoter.sudo("which python3", ignore_status=True).failed:
            cmd = 'yum' if node.distro.is_rhel_like else 'zypper' if node.distro.is_sles else 'apt-get'
            node.remoter.sudo(f'{cmd} install -y python3', retry=3)
```

Reply:
switched to using `install_package()` that does all this dance already.

anyhow nest ternary conditions, isn't more readable then a plain if/else